.github/actions/upload-release-archive/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Upload release archive
```

Comment:
Does this need to in subdirectory?  Or can it just be `.github/actions/upload-release-archive.yml`

Reply:
Unfortunately looks like can't

https://github.com/anuraaga/wabt/actions/runs/5238734587/jobs/9457913174

![image](https://github.com/WebAssembly/wabt/assets/198344/e9ea1c32-a6ab-4e79-bfea-659884e04746)